pkg/flagbuilder/build_flags.go

```diff
@@ -131,6 +131,12 @@ func BuildFlags(options interface{}) (string, error) {
 				flag = fmt.Sprintf("--%s=%s", flagName, vString)
 			}
 
+		case *string:
```

Comment:
Austin i would ask that you add some simple tests just for this code. This is a seriously important piece of logic